README.md

```diff
@@ -787,11 +787,11 @@ public final class BlackjackHand {
     <summary>Kotlin</summary>
 
 ```kotlin
-class BlackjackHand {
-  @Transient private val total: Int
+class BlackjackHand(
```

Comment:
This case should have a default value assigned to it or be converted to a non-constructor var property. Probably something like this

```kotlin
class BlackjackHand(...) {
  @Transient
  private var total: Int = 0
}
```

Reply:
Done.